spacy/symbols.pxd

```diff
@@ -83,6 +83,7 @@ cdef enum symbol_t:
     ENT_TYPE
     HEAD
     SENT_START
+    SENT_END
```

Comment:
Unfortunately the order of the symbols and attributes matters -- it influences which ID will be assigned to that attribute. So we need new attributes to go to the end, and we can only reorder them when we're ready to break model compatibility.

If we insert the `SENT_END` symbol here, the subsequent symbols will be mapped to different values, which could break models.